yarn-error.log

```diff
@@ -0,0 +1,57 @@
+Arguments: 
```

Comment:
you don't need to push this file to the repository. it is a good practice to add it to the .gitignore file.

Reply:
@AlsasaAmmar I see that you resolved this conversation but this file is still in the PR - can you remove it? :)

Reply:
I added it to the gitignore. Should I delete it from here?

Reply:
yesss :) you just want to have this file locally, not in the repository